routes/members.js

```diff
@@ -50,4 +52,29 @@ router.get('/', membersController.getMembers)
 
 router.get('/idle', membersController.getIdleMembers)
 
+/**
+ * @swagger
+ * /members/intro/{username}:
+ *   post:
+ *     summary: Posts details of the recruiter
+ *     tags:
+ *       - Members
+ *     responses:
+ *       200:
+ *         description: Details of the recruiter and the member in which recruiter is interested
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/recruiters'
```

Comment:
I can't find this schema 🤔 
I guess that's remaining to add

Reply:
@Ankita2002-Fr

Reply:
Done!

Reply:
still can't find it
you've to add it in `docs/swaggerDefinition.js`